subsys/bluetooth/services/ots/ots_client.c

```diff
@@ -1115,16 +1113,21 @@ static int oacp_read(struct bt_conn *conn,
 	/* TODO: How do we ensure that the L2CAP is connected in time for the
```

Comment:
We might need to add a small state machine here? I had a related question about not closing the L2CAP connection immediately after a read or write is done.

Reply:
https://github.com/zephyrproject-rtos/zephyr/issues/42647